### ISSUE TITLE
bug: `cosmoshub` does not support `faucet type`: `starship`

### ISSUE THREAD

**Issue by 0xpatrickdev** [cosmology-tech/starship#482]
When using the following settings with the 0.2.2 helm chart:
```
chains:
-  id: cosmoshub-test
    name: cosmoshub
    faucet:
      enabled: true
      type: starship
```

The faucet container in the cosmoshub-test pod fails to start:

```
{"level":"error","ts":1718310048.524163,"caller":"app/distributor.go:286","msg":"send token failed","cmd":"gaiad tx bank send cosmos1vhdew4wqu3tp8l2d55aqcc73aqvr0rr9yk4a0l cosmos1prdhkqcdzz2dqp3qwpmecna6e7faghkkqk2p9p 80000000000stake --chain-id=gaialocal --fees=10000000uatom --keyring-backend=test --gas=auto --gas-adjustment=1.5 --yes --node=http://localhost:26657","error":"err: exit status 1, stderr: Error: rpc error: code = Unknown desc = rpc error: code = Unknown desc = failed to execute message; message index: 0: 0stake is smaller than 80000000000stake: insufficient funds


2024-06-13 16:15:36 err: exit status 1, stderr: Error: rpc error: code = Unknown desc = rpc error: code = Unknown desc = failed to execute message; message index: 0: 0stake is smaller than 80000000000stake: insufficient funds [cosmos/cosmos-sdk@v0.45.16/x/bank/keeper/send.go:186] With gas wanted: '0' and gas used: '60680' : unknown request
```

_The `@cosmjs/faucet` faucet works, but does not support sending tokens to ICA addresses. See this pull request for an upstream fix: https://github.com/cosmos/cosmjs/pull/1593_

**Comment by Anmol1696** [comment]
let me have a look

**Comment by Anmol1696** [comment]
As for `cosmjs` we can wait for the upstream PR to be merged, then we can create a new version of the faucet based on the new version of cosmjs

**Comment by Anmol1696** [comment]
@0xpatrickdev thanks for opening this issue. Fixed the issue in chart version `0.2.4`. Please upgrade and let me know if it works. 

More over, will upgrade the cosmjs version as well, once your PR is merged.
